docs/guides/screenshots/config.json_directives/logging_use_colors_true.png

```diff
@@ -0,0 +1 @@
+
```

Comment:
Blank file?

Reply:
Forgot to remove, used it to create the folder to upload the rest of the pngs